app.js

```diff
@@ -1,6 +1,16 @@
 import config from './config/config';
 import { User, Product} from './models';
+import DirWatcher from './dirWatcher';
+import Importer from './importer';
 
 console.log(config.name);
-const UserObj = new User();
-const ProductObj = new Product();
+const UserObj = new User(),
```

Comment:
Please don't use multiple variable defining, use single instead. This practice aimed to prevent accidentally defining global variables. 
`const a = 'a',`
          `b = 'b'`
         ` c = 'c'  // this variable becomes global due to missed coma in the upper line`
so the best practice is to define every single variable with its own keyword

Reply:
Thanks for the suggestion. Would follow the same going forward.